.github/workflows/node.yml

```diff
@@ -269,3 +271,74 @@ jobs:
                       node/test-report*.html
                       utils/clusters/**
                       benchmarks/results/**
+
+    start-self-hosted-runner:
```

Comment:
This step maybe not needed. Java CI works without it

Reply:
I was referencing the python yaml, wonder why these differences exist?

Reply:
I found that start is no-op, runner is online 24/7 and skipped in java CI.